src/autopas/containers/verletListsCellBased/verletListsCells/VerletListsCells.h

```diff
@@ -115,12 +115,183 @@ class VerletListsCells : public VerletListsLinkedBase<Particle> {
    */
   size_t getNumberOfPartners(const Particle *particle) const { return _neighborList.getNumberOfPartners(particle); }
 
+  /**
+   * Special case of building the neighbor lists in c08 style where all lists that belong to one base step are stored
+   * together.
+   */
+  void rebuildNeighborListsC08() {
+    using namespace utils::ArrayMath::literals;
```

Comment:
I think here an additional AutoPas warning would be nice, if the cellsize factor is smaller than 1.
I guess the actual problematic part is only `VerletListsCellsHelpers::buildC08BaseStep()`, but there you can't access the cellsize factor I think.

Reply:
I think this even warrants an exception since the resulting lists would be potentially wrong -> 52bb083fc
Also, the constructor of VLC currently guarantees `CSF==1`